packages/main/src/plugin/color-registry.ts

```diff
@@ -341,11 +341,26 @@ export class ColorRegistry {
       light: colorPalette.gray[300],
     });
 
+    this.registerColor(`${invCt}card-bg-highlighted`, {
+      dark: colorPalette.charcoal[700],
+      light: colorPalette.gray[200],
+    });
+
+    this.registerColor(`${invCt}card-divide`, {
+      dark: colorPalette.gray[900],
+      light: colorPalette.charcoal[50],
+    });
+
```

Comment:
Look nice to introduce new components

but you can't introduce new components/colors "like that"

the colors are defined in the Podman Desktop design system

And there is not yet these colors or these components

if you need to extract values, it should be defined before the implementation